object/syncer_public_api.go

```diff
@@ -45,12 +44,12 @@ func getEnabledSyncerForOrganization(organization string) (*Syncer, error) {
 			return syncer, nil
 		}
 	}
-	return nil, errors.New("no enabled syncer found")
+	return nil, nil
 }
 
 func AddUserToOriginalDatabase(user *User) error {
 	syncer, err := getEnabledSyncerForOrganization(user.Owner)
-	if err != nil {
+	if syncer == nil {
```

Comment:
Restore the original error handling:

![image](https://github.com/casdoor/casdoor/assets/3787410/e942329b-785e-4bc6-9f79-b331b6358efa)